### ISSUE TITLE
[survival beta] Gold and Silver deposits

### ISSUE THREAD

**Issue by Uncle-Ulty** [tsunamayo/Starship-EVO#5485]
Hi Tsuna,

I started a new survival mode, and everything went well till I needed to look for gold and silver. I took an hour to find them! Maybe these "advanced" ores should be obtained after we build the first overcraft, to make the process less tedious.

cheers

**Comment by Tackietacktack** [comment]
Same one hour and 10 cobalt what I think is rare

**Comment by tsunamayo** [comment]
Yes, also I will add a scanner to know where to go. I am also considering spawning them underground and have the player to explore cave, as you dont need to mine 1000s of them.

**Comment by Uncle-Ulty** [comment]
underground cave would be super cool!
